app/workers/bgs/submit_form674_job.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+
+require 'bgs/form674'
+
+module BGS
+  class SubmitForm674Job
+    class Invalid674Claim < StandardError; end
+    include Sidekiq::Worker
+    include SentryLogging
+
+    # we do individual service retries in lib/bgs/service.rb
+    sidekiq_options retry: false
+
+    def perform(user_uuid, saved_claim_id, vet_info)
+      user = User.find(user_uuid)
+      claim_data = valid_claim_data(saved_claim_id, vet_info)
+
+      BGS::Form674.new(user).submit(claim_data)
+    rescue
+      DependentsApplicationFailureMailer.build(user).deliver_now if user.present?
```

Comment:
Would it be better to deliver_later here in case the DependentsApplicationFailureMailer job fails? Or does sidekiq retry in that case?

Reply:
We can do `deliver_later`. Retries in this job are shut off. We have limited opportunities to submit this form in a time frame, we don't want to retry sending with possible bad data.